simulations/vip-440/ethereum.ts

```diff
@@ -0,0 +1,171 @@
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { forking, testForkedNetworkVipCommands } from "src/vip-framework";
+import { checkIsolatedPoolsComptrollers } from "src/vip-framework/checks/checkIsolatedPoolsComptrollers";
+import { checkRiskParameters } from "src/vip-framework/checks/checkRiskParameters";
+import { checkVToken } from "src/vip-framework/checks/checkVToken";
+import { checkInterestRate } from "src/vip-framework/checks/interestRateModel";
+
+import vip439, { USDS } from "../../vips/vip-439/bscmainnet";
+import vip440, {
+  COMPTROLLER,
+  CONVERSION_INCENTIVE,
+  USDC_PRIME_CONVERTER,
+  USDT_PRIME_CONVERTER,
+  VTOKEN_RECEIVER,
+  WBTC_PRIME_CONVERTER,
+  WETH_PRIME_CONVERTER,
+  XVS_VAULT_CONVERTER,
+  sUSDS,
+  sUSDS_ERC4626_ORACLE,
+  vsUSDS,
+} from "../../vips/vip-440/bscmainnet";
+import ERC4626_ORACLE_ABI from "./abi/ERC4626Oracle.json";
+import POOL_REGISTRY_ABI from "./abi/PoolRegistry.json";
+import RESILIENT_ORACLE_ABI from "./abi/ResilientOracle.json";
+import SINGLE_TOKEN_CONVERTER_ABI from "./abi/SingleTokenConverter.json";
+import COMPTROLLER_ABI from "./abi/comptroller.json";
+import VTOKEN_ABI from "./abi/vToken.json";
+
+const BLOCKS_PER_YEAR = BigNumber.from("2628000");
+const ONE_YEAR = 365 * 24 * 3600;
+
+const { POOL_REGISTRY, NORMAL_TIMELOCK, RESILIENT_ORACLE } = NETWORK_ADDRESSES["ethereum"];
+
+export const newMarkets = {
+  vToken: {
+    address: vsUSDS,
+    name: "Venus sUSDS (Core)",
+    symbol: "vsUSDS_Core",
+    underlying: {
+      address: sUSDS,
+      decimals: 18,
+      symbol: "sUSDS",
+    },
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.73", 18),
+    liquidationThreshold: parseUnits("0.75", 18),
+    supplyCap: parseUnits("30000000", 18),
+    borrowCap: parseUnits("0", 18),
+    reserveFactor: parseUnits("0.1", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("10000", 18),
+    vTokenReceiver: VTOKEN_RECEIVER,
+  },
+  interestRateModel: {
+    address: "0x322072b84434609ff64333A125516055B5B4405F",
+    base: "0",
+    multiplier: "0.15625",
+    jump: "2.5",
+    kink: "0.8",
+  },
+};
+
+forking(21735465, async () => {
+  const provider = ethers.provider;
+  const oracle = new ethers.Contract(RESILIENT_ORACLE, RESILIENT_ORACLE_ABI, provider);
```

Comment:
I think we would not need this oracle as it would not be used, please refer to below comment